src/components/Inputs/ComboBox/ComboBox.tsx

```diff
@@ -185,6 +197,13 @@ export const ComboBox = <T extends ComboBoxOption<T>>(
     }
   };
 
+  const handleOnClear = () => {
+    if ('value' in props) props.onSelect(undefined);
+    else {
+      props.onSelect([], props.values[0]);
```

Comment:
Why do we pass one specific element when clearing multi select?

Reply:
Fixed